### ISSUE TITLE
questions about the training data

### ISSUE THREAD

**Issue by hxx-who** [tzjtatata/Myriad#4]
In the paper, you wrote "Under the zero-shot setting, the expert and Myriad is trained on VisA when testing on MVTec-AD; When applying to VisA, we use all test and training data for training." I was wondering are you training with all the data in the ViSA or just all the good (normal) images in the ViSA?

**Comment by tzjtatata** [comment]
Thank you for the question.&nbsp;
The experts for zero shot and few shot are following their original setting. Because we directly use the pretrained experts. So aprilgan(zero-shot) is pretrained on all data, both normal and abnormal. While patchcore-100%(few shot) is similar to the few shot mode in adgpt，is train-free.&nbsp;
Our model arr trained with normal datas with crop and paste(adding poison noise)，which is as same as NSA.



---Original---
From: ***@***.***&gt;
Date: Sat, Jan 13, 2024 14:16 PM
To: ***@***.***&gt;;
Cc: ***@***.***&gt;;
Subject: [tzjtatata/Myriad] questions about the training data (Issue #4)




 
In the paper, you wrote "Under the zero-shot setting, the expert and Myriad is trained on VisA when testing on MVTec-AD; When applying to VisA, we use all test and training data for training." I was wondering are you training with all the data in the ViSA or just all the good (normal) images in the ViSA?

**Comment by hxx-who** [comment]
> Thank you for the question.&nbsp; The experts for zero shot and few shot are following their original setting. Because we directly use the pretrained experts. So aprilgan(zero-shot) is pretrained on all data, both normal and abnormal. While patchcore-100%(few shot) is similar to the few shot mode in adgpt，is train-free.&nbsp; Our model arr trained with normal datas with crop and paste(adding poison noise)，which is as same as NSA.
> […](#)
> ---Original--- From: ***@***.***&gt; Date: Sat, Jan 13, 2024 14:16 PM To: ***@***.***&gt;; Cc: ***@***.***&gt;; Subject: [tzjtatata/Myriad] questions about the training data (Issue #4) In the paper, you wrote "Under the zero-shot setting, the expert and Myriad is trained on VisA when testing on MVTec-AD; When applying to VisA, we use all test and training data for training." I was wondering are you training with all the data in the ViSA or just all the good (normal) images in the ViSA? — Reply to this email directly, view it on GitHub, or unsubscribe. You are receiving this because you are subscribed to this thread.Message ID: ***@***.***&gt;

But APRILGAN is trained "solely use the test set of the MVTec" according to its paper, why would you say "all data"

**Comment by hxx-who** [comment]
But APRILGAN "solely use the test set of the MVTec" (according to its paper). Why would you say "all test and training data"? Is there anything I miss? thanks!

**Comment by tzjtatata** [comment]
My mistake. We use pretrained aprilgan,just folloilwing the setting in their paper.



---Original---
From: ***@***.***&gt;
Date: Sat, Jan 13, 2024 15:01 PM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [tzjtatata/Myriad] questions about the training data (Issue #4)




 
But APRILGAN "solely use the test set of the MVTec" (according to its paper). Why would you say "all test and training data"? Is there anything I miss? thanks!
